File: app/src/main/java/mhh/com/gotennaapp/api/GetPlacesService.java
package mhh.com.gotennaapp.api;

import java.util.List;
import mhh.com.gotennaapp.model.Place;
import retrofit2.Call;
import retrofit2.http.GET;

public interface GetPlacesService {
    @GET("/development/scripts/get_map_pins.php") Call<List<Place>> getPlaces();
}

File: app/src/main/java/mhh/com/gotennaapp/api/GetPlacesService.kt
package mhh.com.gotennaapp.api

import mhh.com.gotennaapp.model.Place
import retrofit2.Call
import retrofit2.http.GET

interface GetPlacesService {
    @get:GET("/development/scripts/get_map_pins.php")
    val places: Call<List<Place>>
}
